### DISCUSSION TITLE
VSCode IntelliSense is disabled by default

### DISCUSSION THREAD

**Discussion by kof** [remix-run/remix#1591] [Q&A]
Spent quite a bit of time trying to figure out why I get no error reports in VSCode in ts/tsx files. Opening any other project shows them, so my VSCode seems to be configured correctly, but I can't figure out what makes a remix project disable it for good and how to enable it, any hints?

**Comment by pedrogius** [comment]
I'm having the same problem. Working with jsx files and can't get it to work. It does work on other projects, just not on this one.
